app-misc/recoll/recoll-1.32.0.ebuild

```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit linux-info optfeature python-single-r1 qmake-utils systemd
+
+DESCRIPTION="Personal full text search package"
+HOMEPAGE="https://www.lesbonscomptes.com/recoll/"
+SRC_URI="https://www.lesbonscomptes.com/recoll/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="camelcase chm +inotify qt5 session +spell systemd webengine"
+REQUIRED_USE="
+	session? ( inotify )
+	webengine? ( qt5 )
+	${PYTHON_REQUIRED_USE}
+"
+
+DEPEND="
+	dev-libs/xapian:=
+	sys-libs/zlib:=
+	virtual/libiconv
+	chm? (
+		dev-libs/chmlib
+		dev-python/pychm
+	)
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtprintsupport:5
+		dev-qt/qtwidgets:5
+		webengine? ( dev-qt/qtwebengine:5[widgets] )
+	)
+	session? (
+		inotify? (
+			x11-libs/libSM
+			x11-libs/libICE
+			x11-libs/libX11
+		)
+	)
+	spell? ( app-text/aspell )
+	systemd? ( sys-apps/systemd )
+	${PYTHON_DEPS}
+"
+
+BDEPEND="
+	qt5? ( dev-qt/linguist-tools:5 )
+"
+
+RDEPEND="
+	${DEPEND}
+	app-arch/unzip
+"
+
+pkg_pretend() {
+	if has_version "<app-misc/recoll-1.20"; then
+		elog "Installing ${PV} over an 1.19 index is possible,"
+		elog "but there have been small changes in the way"
+		elog "compound words are indexed. So it is best to reset"
+		elog "the index. The best method to reset the index is to"
+		elog "quit all recoll programs and delete the index directory"
+		elog "rm -rf ~/.recoll/xapiandb, then start recoll or recollindex."
+	fi
+}
```

Comment:
I'm going to drop this. Looks like there hasn't been <1.20 in the tree since 2017.